docker-compose.production.yml

```diff
@@ -0,0 +1,16 @@
+##
+## Production server config (wip)
+## You probably want to run:
+## docker-compose -f docker-compose.yml -f docker-compose.infogami-local.yml -f docker-compose.production.yml up -d
+##
+
+version: "3"
+services:
+  web:
+    environment:
+      - PYENV_VERSION=3.8.5
+      - OL_CONFIG=/olsystem/etc/openlibrary.yml
```

Comment:
Is there any option to use threads in later versions of gunicorn (I think this may not have been an option w/ earlier pythons, just something to look into).

There's an @anandology twitter thread about this.

Reply:
that option was there from long back. The liveweb-proxy ran with that configuration when it was setup 7+ years ago.

Reply:
The good thing about process model is that if a request takes too long time, the process will be killed after the timeout. Not sure how that works with threads. You may want to confirm that before making the switch.